src/styles/cylc/_tree.scss

```diff
@@ -0,0 +1,106 @@
+/**
```

Comment:
I have to say I much prefer having code separated into files of the correct extension 👍.